Formula/hledger.rb

```diff
@@ -15,25 +28,19 @@ class Hledger < Formula
     sha256 "22f6bd98c3b593aa5ff55deea85b2c7aa4da9e4d075f1eed05cbca71aefed0ab" => :high_sierra
   end
 
-  depends_on "cabal-install" => :build
-  depends_on "ghc" => :build
+  depends_on "haskell-stack" => :build
+  # Use brew's ghc, assuming it's compatible, instead of letting stack fetch it. Any faster ?
+  # Seems to be failing, maybe from ghc/snapshot minor version mismatch.
+  # depends_ on "ghc" => :build
```

Comment:
It should really use all the brew parts it can.

Reply:
Using brew's ghc will make it more inclined to break over time, but should be ok for now; I'll do that then.